Application/EdFi.Ods.AdminApi/Infrastructure/Extensions/QueryExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using System.Linq.Expressions;
+
+namespace EdFi.Ods.AdminApi.Infrastructure.Extensions
```

Comment:
Use the namespace without braces, sames as Application/EdFi.Ods.AdminApi/Infrastructure/Database/Queries/GetVendorsQuery.cs

Example:
`namespace EdFi.Ods.AdminApi.Infrastructure.Database.Queries;`